src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/VirtualNetworkRuleTest.ps1

```diff
@@ -0,0 +1,165 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+	.SYNOPSIS
+	Tests creating and updating a virtual network rule
+#>
+function Test-CreateAndUpdateVirtualNetworkRule
+{
+	# Setup
+	$location = "East US 2 EUAP"
+	$rg = Create-ResourceGroupForTest $location 	
+	$server = Create-ServerForTest $rg $location
+
+	$virtualNetworkRuleName = Get-VirtualNetworkRuleName
+
+	$vnetName1 = "vnet1"
+	$virtualNetwork1 = CreateAndGetVirtualNetwork $rg $vnetName1 $location
+	$virtualNetworkSubnetId1 = $virtualNetwork1.Subnets[0].Id
+
+	$vnetName2 = "vnet2"
+	$virtualNetwork2 = CreateAndGetVirtualNetwork $rg $vnetName2 $location
+	$virtualNetworkSubnetId2 = $virtualNetwork2.Subnets[0].Id
+
+	try
+	{
+		# Create rule
+		$virtualNetworkRule = New-AzureRmSqlServerVirtualNetworkRule -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName -VirtualNetworkRuleName $virtualNetworkRuleName -VirtualNetworkSubnetId $virtualNetworkSubnetId1
```

Comment:
please make these lines narrower to make it easier to review.  You can use the ` character to have a command span multiple lines.

Reply:
something like this would work?
New-AzureRmSqlServerVirtualNetworkRule -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName 
` -VirtualNetworkRuleName $virtualNetworkRuleName -VirtualNetworkSubnetId $virtualNetworkSubnetId1